mesh/silvair-io.c

```diff
@@ -132,13 +133,39 @@ struct silvair_keep_alive_cmd_pld {
 } __packed;
 
 
+static bool buffered_write(struct l_io *l_io, void *user_data)
+{
+    struct silvair_io *io = user_data;
+
+    (void)l_ringbuf_write(io->l_out_ringbuf, l_io_get_fd(l_io));
+
+    return (l_ringbuf_len(io->l_out_ringbuf) != 0);
```

Comment:
No need to compare against `0`. Yes, I know we have a different coding style than firmware ;)